cypress/e2e/OrangeHRM/approveLeaveRequest.ts

```diff
@@ -0,0 +1,37 @@
+import ApproveRequest from "../../support/helpers/approveRequestHelper";
+import loginPage from "../../support/pageObjects/loginPage";
+import LogoutPage from "../../support/pageObjects/logoutFunctionality";
+import SearchInTable from "../../support/pageObjects/assertionForScheduled";
+const loginObj: loginPage = new loginPage();
+
+let id: any;
+let leaveId: any;
+let userName: string;
+describe("OrangeHRM ", () => {
+  beforeEach(function () {
+    cy.visit("https://opensource-demo.orangehrmlive.com/");
```

Comment:
move this to the command itself so no need to have it each time you do the login